src/app/views/common/lazy-loader/component-registry/index.tsx

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable max-len */
+import { lazy } from 'react';
+import { IPermissionProps } from '../../../../../types/permissions';
+import { SuspenseLoader } from '../suspense-loader/SuspenseLoader';
+
+const LazyPermission = lazy(() =>
+/* webpackChunkName: "permissions" */ import('../../../query-runner/request/permissions'));
```

Comment:
The webpackChunkName magic comment needs to be inside the import() function call to ensure that the chunk actually gets the name you want it to